messages/run.js

```diff
@@ -0,0 +1,37 @@
+module.exports = {
+  "commandDescription": "Evaluate a selection of rules against a specified codebase.",
+  "flags": {
+    "rulenameDescription": "[Description of 'rulename' parameter]",                   // TODO: Change this once the flag is implemented.
+    "categoryDescription": "One or more categories of rules to run. Multiple values can be specified as a comma-separated list.",
+    "rulesetDescription": "One or more rulesets to run. Multiple values can be specified as a comma-separated list.",
+    "severityDescription": "[Description of 'severity' parameter]",                   // TODO: Change this once the flag is implemented.
+    "excluderuleDescription": "[Description of 'exclude-rule' parameter]",            // TODO: Change this once the flag is implemented.
+    "orgDescription": "[Description of 'org' parameter]",                             // TODO: Change this once the flag is implemented.
+    "suppresswarningsDescription": "[Description of 'suppress-warnings' parameter]",  // TODO: Change this once the flag is implemented.
+    "targetDescription": "One or more filepaths or glob patterns to run rules against.",
+    "formatDescription": "Specifies result format, and causes results to be written directly to stdout.",
+    "outfileDescription": "A file to which the results should be written."
+  },
+  "validations": {
+    "mustTargetSomething": "Please specify a target for the rules using --target.", // TODO: Once --org is implemented, rewrite this message.
+    "mustSpecifyOutput": "Please specify an output through either --format or --outfile.",
+    "outfileMustBeValid": "--outfile must be a well-formed filepath.",
+    "outfileMustBeSupportedType": "--outfile must be of a supported type. Current options are .xml and .csv."
+  },
+  "output": {
+    "noViolationsDetected": "No rule violations found."
+  },
+  "examples": `Invoking without specifying any rules causes all rules to be run.
+  E.g., $ sfdx scanner:run --format xml --target "somefile.js"
+    Evaluates all rules against somefile.js.
+    
+Specifying multiple categories or rulesets is treated as a logical OR.
+  E.g., $ sfdx scanner:run --format xml --target "somefile.js" --category "Design,Best Practices" --ruleset "Braces"
+    Evaluates all rules in the Design and Best Practices categories, and all rules in the Braces ruleset.
+    
+Wrap globs in quotes. Use double-quotes in Windows and single-quotes in Unix.
```

Comment:
Not that I'm aware of. I've tested in a few different environments:
- Windows command prompt: If you use single-quotes, you get a bizarre error about the file not existing, even if you're just using the straight filepath. It looks like this is just a known weird quirk that there's no way around.
- Windows powershell: Both single- and double-quotes work fine here.
- OSX Terminal: If you use double-quotes and your glob has a `!`, the `!` gets misinterpreted as part of a command instead of a string, and you get a funky error. Again, no way around this that I could find.